### PR TITLE
Update intro.md

### PR DESCRIPTION
**Pull request by iamfortune**

### CODE REVIEW

File: content/management/v1/core-resources/deployments/intro.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ The endpoint /v1/spaces/:space_id/deployments can be used to deploy branches.
 
 You can read more about branches [here](https://www.storyblok.com/docs/setup-branches-and-releases#pipeline-stage).
 
+It's important to note that stories created via the Management API would be created in the default branch. However, using additional parameters, the content can be published to specific branches (or pipeline stages, as they referred to them).
```
Comment (manuelschroederdev) [inline content/management/v1/core-resources/deployments/intro.md:9]: Please replace it with:

Stories created via the Management API are created in the default branch (Preview). Subsequently, they can be released to other branches (e.g. Staging, Production) by creating a branch deployment.

**Review by manuelschroederdev — COMMENTED** [review]
Hey @iamfortune can you please make these changes

File: content/management/v1/core-resources/deployments/intro.md (hunk 1 above)
Comment (roberto-butti) [reply content/management/v1/core-resources/deployments/intro.md:9]: I'd like to suggest adding the information "according to the "source of sync" option". @manuelschroederdev @iamfortune  what do you think : 

Stories created via the Management API are created in the default branch (Preview). Subsequently, they can be released to other branches (e.g. Staging, Production **according to the "source of sync" option**) by creating a branch deployment.

Comment (manuelschroederdev) [reply content/management/v1/core-resources/deployments/intro.md:9]: Yes, I like that. Can you please update the PR accordingly @iamfortune?

Comment (iamfortune) [reply content/management/v1/core-resources/deployments/intro.md]: Done 👍🏽

**Pull request merged**